P12.py

```diff
@@ -3,12 +3,27 @@
 import threading
 import math
 
+class ThreadPool:

```

Comment:
https://docs.python.org/3/library/multiprocessing.html

Reply:
It won't work in class. It needs function outside. And it will make problems with variables (self.files._list, self.duplicates ...) in my situation. So, I got it that it is possible to make it with Pool class from library)